### PR TITLE
Fix for improper y axis scaling in LineChart

### LINKED ISSUE #26383: LineChart Y axis are not scaled for the values between 0 and 1
When using LineChart component I would like the plots to be scaled based on the min and maximum value for Y axis.
Unfortunately, when I am using the component parameters ` yMaxValue=0.9` and `yMinValue=0.1`, the Y axis of the plot are showing the values from 100m to 1.1, which is definitely not my intention. Please check the image below.

### Environment Information

- **Package version(s)**: "@fluentui/react-charting": "5.12.19",
- **Browser and OS versions**: Edge

### Describe the issue:

When using LineChart component I would like the plots to be scaled based on the min and maximum value for Y axis.
Unfortunately, when I am using the component parameters ` yMaxValue=0.9` and `yMinValue=0.1`, the Y axis of the plot are showing the values from 100m to 1.1, which is definitely not my intention. 
My guess is that there is a strange behavior when dealing with values between 0 and 1.

line_bug_from_0_to_1

#### Expected behavior:

The Y axis are scaled according to the float values that are given, namely 0.1 and 0.9 or 0.0015 and 0.02.
and there are also ticks shown in between those values.

### PR DESCRIPTION
**Pull request by Shubhabrata08**
## Previous Behavior

image

## New Behavior

image

## Related Issue(s)

- Fixes #26383

### CODE REVIEW

**Comment by AtishayMsft** [comment]
az run

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/af50ddcf8a6f9830d3627e9e1f0e45cb8f76277c'>af50ddcf8a6f9830d3627e9e1f0e45cb8f76277c</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/utilities/utilities.ts [hunk 1]
```diff
@@ -304,7 +304,8 @@ export function createStringXAxis(
  * @returns {number[]}
  */
 export function prepareDatapoints(maxVal: number, minVal: number, splitInto: number): number[] {
```
Comment (AtishayMsft) [inline packages/react-charting/src/utilities/utilities.ts:306]: Add a change file for this change

**Review by AtishayMsft — APPROVED** [review]

File: packages/react-charting/src/utilities/utilities.ts (hunk 1 above)
Comment (Shubhabrata08) [reply packages/react-charting/src/utilities/utilities.ts:306]: Added changefile in latest push

**Pull request merged**